docs/source/deployment/tgi.rst

```diff
@@ -0,0 +1,106 @@
+TGI
+=====================
+
+Hugging Face's Text Generation Inference (TGI) is a production-ready framework specifically designed for deploying and serving large language models (LLMs) for text generation tasks. It offers a seamless deployment experience, powered by a robust set of features:
+
+* **Speculative Decoding:**  Accelerates generation speeds.
+* **Tensor Parallelism:**  Enables efficient deployment across multiple GPUs.
+* **Token Streaming:**  Allows for the continuous generation of text.
+* **Versatile Device Support:**  Works seamlessly with AMD, Gaudi, AWS Inferentia, and Google TPU.
+
+
+Installation
+-----------------
+
+The easiest way to use TGI is via the TGI docker image, but it's possible to run it locally via Conda. Please refer this doc for instructions on the installation: `https://github.com/huggingface/text-generation-inference?tab=readme-ov-file#local-install`
+
+
+Deploy Qwen with TGI
+-----------------------
+
+Find a model from qwen2 collection and get the model path: `https://huggingface.co/collections/Qwen/qwen2-6659360b33528ced941e557f`
+
+You can either run the model via CLI or via Docker
+
+
+.. code:: bash
```

Comment:
I would suggest to also include a Python example

Reply:
Not sure how to start TGI with python but added a Python client example.